app/src/main/java/org/sopt/carrot/data/model/response/SearchResponse.kt

```diff
@@ -0,0 +1,20 @@
+package org.sopt.carrot.data.model.response
```

Comment:
p3: 이제보니 초기 세팅에서 저도 놓친 부분인데 data 레이어에서는 model보단 dto가 더 적합하다고 생각해서 한번 조율해야할 거같습니다!! @angryPodo @hyoeunjoo 
```suggestion
package org.sopt.carrot.data.dto.response
```

Reply:
엥 나 왜저랬지 맞습니당;; 꼼꼼핑 제 브랜치에서 나중에 패키지명 수정할게요!